deploy/alerts/alerts.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: monitoring.coreos.com/v1
+apiVersion: monitoring.rhobs/v1
```

Comment:
This is probably going to require documentation to address migration to another Operator in the future?

Reply:
Ultimately, now might be a good time to start managing these in the operator instead of a documented `oc apply -f`